getnyumba/node-express-api
# #166730711 register new user

#### Description
- enable a new user to register
- send verification email to user
- verify user account
- write unit tests
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test

Give instructions on how reviewers can verify your work

- fetch branch using `git fetch ft-user-registration-166730711 && git checkout ft-user-registration-166730711 `
- start dev server using `yarn start:dev`
- Hit the endpoint using a get method `http://127.0.0.1:8000/api/v1/users/register`

#### Any background context you want to add
```Request body below.```
```{
	"first_name":"fname",
	"second_name":"sname",
	"phone_number":"0703640000",
	"username":"username",
	"email":"youremail@email.com",
	"password":"password",
	"confirmPassword":"password"
}
```
[Finishes #166730711]
